documentation/src/docs/asciidoc/release-notes-5.0.0-M4.adoc

```diff
@@ -82,6 +81,8 @@ on GitHub.
 ===== Deprecations and Breaking Changes
 
 * Removed deprecated `Assertions.expectThrows()` method in favor of `Assertions.assertThrows()`.
+* ExtensionContext.Namespaces with the same parts but in a different order are no longer considered
```

Comment:
Please frame `ExtensionContext.Namespace` with " ` " characters. Everything else looks good to me.